app/controllers/api/red_hat_migration_analytics_controller.rb

```diff
@@ -46,18 +64,35 @@ def find_provider_ids(type)
     end
 
     class << self
-      def parse_manifest
-        default_manifest_path = Cfme::MigrationAnalytics::Engine.root.join("config", "default-manifest.json")
-        user_manifest_path    = Pathname.new("/opt/rh/cfme-migration_analytics/manifest.json")
-        manifest_path         = user_manifest_path.exist? ? user_manifest_path : default_manifest_path
 
-        manifest = Vmdb::Settings.filter_passwords!(load_manifest(manifest_path))
+      def default_manifest_path
+        @default_manifest_path ||= Cfme::MigrationAnalytics::Engine.root.join("config", "default-manifest.json")
+      end
+
+      def user_manifest_path
+        @user_manifest_path ||= Pathname.new("/opt/rh/cfme-migration_analytics/manifest.json")
```

Comment:
Should this be in the root `/opt/rh/cfme-migration_analytics`, or would `/opt/rh/cfme-migration_analytics/config/` be better?

Reply:
Or ignore this...didn't see there was discussion on it, and I was thinking it was actually writing into the gem's location.